bin/include/jvmdefaults.sh

```diff
@@ -103,3 +103,9 @@ getJavaSpecificOpts() {
 
   echo $value
 }
+# Reflective access options
+addReflectiveAccessOptions() { 
```

Comment:
I think the exact list of options should be based on the JVM version. For example, using these opts with JVM8 leads to 
```
Unrecognized option: --add-exports=java.base/jdk.internal.misc=ALL-UNNAMED
Error: Could not create the Java Virtual Machine.
Error: A fatal exception has occurred. Program will exit.
```

Perhaps, I am missing something.

Reply:
Thank you for highlighting the compatibility issues with Java 8. I realize now that my updates to the jvmdefaults.sh script included Java 9+ specific options without checks for Java version compatibility. I will revise the script to dynamically adjust the JVM options based on the version it detects. 

Yes as recommended I will also conduct testing across these Java versions to confirm that the script functions correctly in each environment. 

Once you've confirmed my approach regarding the initial comment of the PR, I will make all necessary changes and conduct all tests at once.